Android/GoogleDagger/source/Dagger/Dagger.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could you make this `<Project Sdk="MSBuild.Sdk.Extras/2.0.54">` to use the newer sdk extras package?  Also see some comments below to remove unnecessary bits after this change...